store_data.rb

```diff
@@ -0,0 +1,60 @@
+def store_data(app)
+  store_people(app)
+  store_books(app)
+  store_rentals(app)
+end
+
+def store_people(app)
+  people = []
+  storage = Set.new
```

Comment:
- [ ] Hey guys @abdozayan12 @hmunish, It seems like I'm getting the same error as well I don't know how it's working on your machine but you can add `require 'set'` to properly import the set library and make the Set class available for use in your code. I've tried this and it's working fine on my end.
  ![image](https://github.com/abdozayan12/OOP-school-library/assets/104968236/93d7d0b0-399b-4321-aa04-9971af6b6161)